x/mint/proto/mint/v3/events.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package mint.v3;
```

Comment:
We at minimum have to update `ConsensusVersion = 3` in `x/mint` when we do this. I followed up with this in my commit

Reply:
@relyt29 @guilherme-brandao Anything else we have to do here to upgrade the module. No migration needed I believe in this case.